tslearn/docs/examples/plot_sdtw.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+"""
+sDTW multi path matching
+========================
+
+This example illustrates how subsequent DTW can be used to find multiple
+matches of a sequence in a longer sequence.
+
+A potential usecase is to identify the occurrence of certain events in
+continuous sensor signals. As one example Barth et al. [1] used this method
+to find stride in sensor recordings of gait
+
+The example demonstrates the use of the functions
+`accumulated_subsequence_cost_matrix` and `calculate_subsequence_path`
+to manually calculate warping paths from multiple potential alignments. If
+you are only interested in finding the optimal alignment, you can directly use
+`dtw_subsequence_path`.
+
+[1] Barth, et al. (2013). Subsequence dynamic time warping as a method for
```

Comment:
Looks like the reference here does not render nicely. From `sklearn` docs (eg [here](https://scikit-learn.org/stable/auto_examples/applications/plot_species_distribution_modeling.html#sphx-glr-auto-examples-applications-plot-species-distribution-modeling-py), it seems that using bullet lists for references is more convenient.

Reply:
This does not seem to be consistent. At other instances they used the same citation style I used (e.g. [here](https://github.com/scikit-learn/scikit-learn/blob/master/examples/covariance/plot_mahalanobis_distances.py)

Reply:
You are perfectly right. My main concern in fact was about the obtained formatting: in the case you mention, the first example is not properly rendered (the name of the journal starts in bold on the first line and then continues without bold on the second line) while formatting of the second item is OK (title + authors in bold, the rest in non-bold). If you can find a way to have some reasonable rendering in your case that would be perfect.

Reply:
I will have a look and make sure it looks good in the hml output :)

Reply:
OK, great! If you build the doc locally, you may have to uncomment the following line (at least I have to on my side :)

https://github.com/rtavenar/tslearn/blob/master/tslearn/docs/conf.py#L23